TestON/tests/MultiProd/MultiProd.py

```diff
@@ -53,18 +53,19 @@ def CASE1( self, main ):
         removeLogResult = main.ONOSbench.onosRemoveRaftLogs()
 
         main.step( "Git checkout, pull and get version" )
-        #main.ONOSbench.gitCheckout( "master" )
+        main.ONOSbench.gitCheckout( "master" )
         gitPullResult = main.ONOSbench.gitPull()
         main.log.info( "git_pull_result = " + str( gitPullResult ))
         versionResult = main.ONOSbench.getVersion( report=True )
 
-        if gitPullResult == 1:
+        packageResult = main.TRUE
+        if gitPullResult == 100:
```

Comment:
That was my purpose I do not want them to pull in different commit when
running in jenkins nightly, So the first test, Single 1.0, in this case
will pull the latest master and they will all run in that commit.

On Wed, Jun 10, 2015 at 1:16 PM, Jon Hall <notifications@github.com> wrote:

> In TestON/tests/MultiProd/MultiProd.py
> <https://github.com/opennetworkinglab/ONLabTest/pull/131#discussion_r32161263>
> :
>
> >          gitPullResult = main.ONOSbench.gitPull()
> >          main.log.info( "git_pull_result = " + str( gitPullResult ))
> >          versionResult = main.ONOSbench.getVersion( report=True )
> >
> > -        if gitPullResult == 1:
> > +        packageResult = main.TRUE
> > +        if gitPullResult == 100:
>
> Is this ever true?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/opennetworkinglab/ONLabTest/pull/131/files#r32161263>.
>